nemo_aligner/models/nlp/gpt/megatron_gpt_regression_reward_model.py

```diff
@@ -114,6 +114,47 @@ def loss_func(output_tensor):
         return fwd_output_and_loss_func
 
     def loss_func(self, output_tensor, label_tensor):
+        loss_func_name = self.cfg.get("loss_func", "regression")
+
+        if loss_func_name == "regression":
+            return self.reg_loss_func(output_tensor, label_tensor)
+        elif loss_func_name.endswith("bt"):
+            return self.bt_loss_func(output_tensor, label_tensor)
+        else:
+            raise ValueError("only accepted values for loss_func are regression, regular_bt, margin_bt and scaled_bt")
```

Comment:
maybe add a check for these types explicitly? something like 
`assert loss_func_name in ["regression", "regular_bt", "margin_bt" and "scaled_bt"]`

Reply:
sure